packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-remote-media-item-node.js

```diff
@@ -255,110 +256,138 @@ export const createRemoteMediaItemNode = async ({
   const fetchState = {
     shouldBail: false,
   }
-  // Otherwise we need to download it
-  const remoteFileNode = await retry(
-    async () => {
-      if (fetchState.shouldBail) {
-        failedImageUrls.add(mediaItemUrl)
-        return null
-      }
 
-      const createFileNodeRequirements = {
-        parentNodeId: mediaItemNode.id,
-        store: gatsbyStore,
-        cache,
-        createNode,
-        createNodeId,
-        reporter,
-      }
+  const createFileNodeRequirements = {
+    parentNodeId: mediaItemNode.id,
+    store: gatsbyStore,
+    cache,
+    createNode,
+    createNodeId,
+    reporter,
+  }
 
-      if (hardCacheMediaFiles) {
-        // check for file in .wordpress-cache/wp-content
-        // if it exists, use that to create a node from instead of
-        // fetching from wp
-        try {
-          const buffer = await fs.readFile(hardCachedFilePath)
-          const node = await createFileNodeFromBuffer({
-            buffer,
-            name: title,
-            ext: path.extname(mediaItemUrl),
-            ...createFileNodeRequirements,
-          })
+  let imageNode
 
-          if (node) {
-            return node
-          }
-        } catch (e) {
-          // ignore errors, we'll download the image below if it doesn't exist
+  const shouldFetchImage = fileSize > maxFileSizeBytes
```

Comment:
I think this is a little confusing, most likely just because the variable name.

The variable is called shouldFetchImage, but if that is true, the comment above the if statement says "don't fetch the remote file".

Reply:
Ahh dang it, I renamed this and then stashed the changes before I committed them :joy: Just fixed it!